### DISCUSSION TITLE
Redis commands to implement (upto 6.2)

### DISCUSSION THREAD

**Discussion by sazzad16** [xetorthio/jedis#2379] [General]
- [ ] CLIENT TRACKINGINFO subcommand (#7309)
- [ ] RESET command that resets client connection state (#7982)
- [ ] Add hostname support to Sentinel (#8282)
- [x] LOLWUT command. http://antirez.com/news/123
- [x] XINFO <key> special form was removed, now XINFO STREAM <key> must be used
- [x] XINFO STREAM FULL, a new subcommand to get the whole stream state
- [x] GEOSEARCH/GEOSEARCHSTORE commands for bounding box spatial queries (#8094)
- [x] ANY argument to GEOSEARCH and GEORADIUS (#8259)
- [x] REV, BYLEX and BYSCORE arguments to ZRANGE, and the ZRANGESTORE command (#7844)
- [x] exclusive range query to XRANGE and XREVRANGE (#8072)
- [x] exclusive range query to XPENDING (#8130)
- [x] REPLICAOF command
- [x] local address to CLIENT LIST, and a CLIENT KILL filter. (#7913)
- [x] ACL patterns for Pub/Sub channels (#7993)
- [x] Support ACL for Sentinel mode (#7888)

**Comment by gkorland** [comment]
https://github.com/redis/jedis/projects/1

**Comment by sazzad16** [comment]
Similar list: https://github.com/phpredis/phpredis/issues/1894

**Comment by zeekling** [comment]
here is commands supported in redis7:

https://github.com/redis/jedis/issues/2795  https://github.com/redis/jedis/issues/2794  https://github.com/redis/jedis/issues/2793  https://github.com/redis/jedis/issues/2792 https://github.com/redis/jedis/issues/2791 https://github.com/redis/jedis/issues/2789 


@sazzad16

**Reply by sazzad16** [reply]
@zeekling https://github.com/redis/jedis/discussions/2807

**Reply by zeekling** [reply]
> @zeekling #2807

ok
